### DISCUSSION TITLE
Call of duty cold war Unlock all

### DISCUSSION THREAD

**Discussion by xNakuty** [community/community#128157] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,
Im actually looking for an unlock all for cold war, i just buy the game on pc (i was on ps5) and 5 operators (that i really like) was dissapear ?
I have contacted CoD for that issue, but they don't give a f***...
So im looking for an unlock all (not soft unlock all) i don't want to re buy operator for a dead game.
Thanks in advence

**Comment by ebndev** [comment]
Hi @xNakuty,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on gaming. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. 

Thanks for understanding!
